language/Pattern-Matching.md

```diff
@@ -33,6 +33,7 @@ The following pattern types are supported:
 - Record patterns
 - Named patterns
 - Guards
+- Pattern guards
```

Comment:
Ah, sorry I misunderstood. I personally cannot think of a better way to integrate them, though. Would you like a note afterward like "Guards and pattern guards are also supported"?

I could also hyperlink them, that might be nice …